shell/browser/native_window_views.h

```diff
@@ -289,6 +289,9 @@ class NativeWindowViews : public NativeWindow,
   HWND legacy_window_ = nullptr;
   bool layered_ = false;
 
+  // this value is determined when the window is created.
```

Comment:
nit: This starting word should have its first letter capitalized to match the rest of the comments in the file

Reply:
done.